support-frontend/assets/pages/showcase/components/hero.jsx

```diff
@@ -2,68 +2,98 @@
 
 import React from 'react';
 
-import { HeroWrapper } from 'components/productPage/productPageHero/productPageHero';
+import { GBPCountries, AUDCountries, Canada, EURCountries, International, NZDCountries, UnitedStates } from 'helpers/internationalisation/countryGroup';
 import GridImage from 'components/gridImage/gridImage';
-
+import { regionalContent } from './regionalContent';
 import HeroImg from './hero.svg';
-
 import './hero.scss';
 
-export default function BreakingHeadlines() {
+const Caption = (props: { className: string, captionTitle: String, captionText: String }) => (
+  <div className={props.className}>
+    <figcaption className="showcase-hero__caption">
+      <strong>{props.captionTitle}</strong>
+      <p>{props.captionText}</p>
+    </figcaption>
+  </div>
+);
+
+const getCountrySelector = (country: String) => {
+  switch (country) {
+    case GBPCountries:
+    case EURCountries:
+    case International:
+    case Canada:
+    case NZDCountries:
+      return 'ukContent';
```

Comment:
I would make this function return a `CountryGroupId` rather than a string and then key the `regionalContent` object with `CountryGroupId`s as well

Reply:
Done